### PR TITLE
[Feature](bangc-ops): add ms_deform_attn_forward proto

### PR DESCRIPTION
**Pull request by Unireverse**
Add ms_deform_attn_forward proto

### CODE REVIEW

File: mlu_op_test.proto [hunk 1]
```diff
@@ -835,3 +836,7 @@ message RoiAlignBackwardParam {
   optional int32 version = 5 [default = 0];
 }
```
Comment (Wickyzheng) [inline mlu_op_test.proto:838]: // param to call mluOpMsDeformAttnForwardParam()

File: mlu_op_test.proto [hunk 2]
```diff
@@ -295,6 +295,7 @@
    optional MoeDispatchBackwardGateParam  moe_dispatch_backward_gate_param   = 135246;  // param
    optional MatMulParam            matmul_param                  = 93;   // MatMulParam
    optional RoiAlignBackwardParam roi_align_backward_param       = 4427;  //param
+   optional MsDeformAttnForwardParam ms_deform_attn_forward_param = 16005; // MsDeformAttnForward
```
Comment (Wickyzheng) [inline mlu_op_test.proto:298]: // MsDeformAttnForwardParam

File: mlu_op_test.proto [hunk 3]
```diff
@@ -295,6 +295,7 @@
    optional MoeDispatchBackwardGateParam  moe_dispatch_backward_gate_param   = 135246;  // param
    optional MatMulParam            matmul_param                  = 93;   // MatMulParam
    optional RoiAlignBackwardParam roi_align_backward_param       = 4427;  //param
```
Comment (duzekunKTH) [inline mlu_op_test.proto:297]: 能不能顺便把这一行改了。这个注释看着好难受

**Review by Wickyzheng — APPROVED** [review]
lgtm

**Pull request merged**